### DISCUSSION TITLE
showing flightplan and datas in EFB

### DISCUSSION THREAD

**Discussion by KaiBVG** [flybywiresim/aircraft#3332] [Help] [answered]
Hi,
just downloaded the latest development mod with custom FBW and loaded the flightplan in the CDU via AOC-menu created in SimBrief. But I can't import the values and the plan into the EFB. Pushing the SimBrief-button has no influence. Do I have to do something else to get them into the EFB (I watched this function in a video already). Thanks!

**Comment by tracernz** [comment, marked as answer]
Set your simbrief username on the company page.

**Reply by KaiBVG** [reply]
Thanks, just did it. But it doesn't work. Do I have to try the SimBrief ID instead? And I didn't find a way to save my inputted name. Does it save automatically?

**Reply by drforweb** [reply]
Same for me:  I enter the username but I cannot confirm it ...

**Reply by KaiBVG** [reply]
I solved it: the SimBrief-name has to be the login-name in SimBrief, not the pilot name! LOL, now it works!
